homeassistant/components/fritzbox/const.py

```diff
@@ -24,3 +44,53 @@
 LOGGER: Final[logging.Logger] = logging.getLogger(__package__)
 
 PLATFORMS: Final[list[str]] = ["binary_sensor", "climate", "switch", "sensor"]
+
+BINARY_SENSOR_TYPES: Final[tuple[FritzBinarySensorEntityDescription, ...]] = (
+    FritzBinarySensorEntityDescription(
```

Comment:
Please define the platform specific entity descriptions in each platform where they are used. They are platform details.